custom_components/adaptive_lighting/switch.py

```diff
@@ -343,33 +429,103 @@ async def async_setup_entry(
         update_before_add=True,
     )
 
+    @callback
+    async def handle_apply(service_call: ServiceCall):
+        """Handle the entity service apply."""
+        _LOGGER.debug(
+            "Called 'adaptive_lighting.apply' service with '%s'",
+            service_call.data,
+        )
+        these_switches = data = None
```

Comment:
This assignment seems unnecessary.

Reply:
It needs to initialize somehow.

Reply:
But you are literally assigning them in the very next line 😅😜

Reply:
Wow so I am. I guess I can't keep up with the amount of code I've moved around.

That can be removed.